src/test/kotlin/org/rust/ide/intentions/IfLetToMatchIntentionTest.kt

```diff
@@ -24,15 +25,17 @@ class IfLetToMatchIntentionTest : RsIntentionTestBase(IfLetToMatchIntention()) {
     fn main() {
         if let Some(val) = x {
 
-        } else {
-            /*caret*/
+        } else {/*caret*/
+            println!("it work")
         }
     }
     """, """
     fn main() {
         match x {
             Some(val) => {},
-            _ => {}
```

Comment:
At the moment we can't check exhaustiveness of pattern matching, but maybe it will be better to handle `Option` and `Result` in a special way for now?

Reply:
```rust
Some(x) => {},
None    => {}
```
for `Option` and
```rust
Ok(x) => {},
Err(e) => {}
```
for `Result`?

Reply:
Yes.

Reply:
Done.